lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/mohamedibrahim/UserAlreadyRegistered.java

```diff
@@ -0,0 +1,7 @@
+package com.codedifferently.lesson9.mohamedibrahim;
+
+public class UserAlreadyRegistered extends RuntimeException {
```

Comment:
It is conventional to end the name of a custom exception with the word `Exception`. Please follow this convention.